challenges.yml

```
@@ -65,3 +70,118 @@ stimulus_controllers_convention:
         c: |
           It will connect to the `api_manager.js` file
       correct: b
+stimulus_targets:
+  question: |
+    What would you have to change to this controller to be able to get an array
+    of all the matched targets instead of just one?
+    ```js
+    export default class extends Controller {
+        static targets = ['item'];
+
+        connect() {
+            this.itemTarget.forEach(() => {
```

```suggestion
            this.itemTarget.forEach((itemElement) => {
```

... it's just more realistic